installer/install.sh

```
@@ -0,0 +1,635 @@
+
+                    # Start and enable Docker on boot
+                    sudo systemctl enable --now docker
+
+                    # Add your user to docker group (optional, allows running docker without sudo)
+                    sudo usermod -aG docker $USER
+                    echo "Docker installation complete. Please log out and log back in for group changes to take effect."
+                fi
+                
+                echo "Setting up MythicalDash-v3..."
+                sudo mkdir -p /var/www/mythicaldash-v3
+                cd /var/www/mythicaldash-v3
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)